Firmware/mesh_bed_calibration.cpp

```diff
@@ -3125,11 +3125,7 @@ void mbl_settings_init() {
 //magnet elimination: use aaproximate Z-coordinate instead of measured values for points which are near magnets
 	eeprom_init_default_byte((uint8_t*)EEPROM_MBL_MAGNET_ELIMINATION, 1);
 	eeprom_init_default_byte((uint8_t*)EEPROM_MBL_POINTS_NR, 3);
-	mbl_z_probe_nr = eeprom_read_byte((uint8_t*)EEPROM_MBL_PROBE_NR);
-	if (mbl_z_probe_nr == 0xFF) {
-		mbl_z_probe_nr = 3;
-		eeprom_update_byte((uint8_t*)EEPROM_MBL_PROBE_NR, mbl_z_probe_nr);
-	}
+	mbl_z_probe_nr = eeprom_init_default_byte((uint8_t*)EEPROM_MBL_PROBE_NR, 3);
```

Comment:
:thinking: let's name `eeprom_init_default_byte` something like `eeprom_read_byte_or_default`. It's semantics has evolved from a simple init if unset.

Reply:
I don't really see the problem with eeprom_init_default_byte. Init here could be taken as both initializing the eeprom cell and as initializing the cached value (assignment). I find byte_or_default more confusing tbh.

Reply:
ok